src/Themes/ReconstructionCard.scss

```
@@ -0,0 +1,40 @@
+.color-box-com {
+  display: flex;
+  justify-content: space-between;
+  border-radius: 15px;
+  position: relative;
+  cursor: pointer;
+}
+.recent-model {
+  width: 360px;
+  height: 234px;
+  background-color: lightblue;
+  border-radius: 12px;
+}
+
+.card_details {
```

Use `-` instead of `_`
```suggestion
.card-details {
```